Lib/fontTools/ttLib/tables/otData.py

```diff
@@ -2,1666 +2,5463 @@
 from fontTools.misc.py23 import *
 
 otData = [
-
```

Comment:
I see you have run `black` on otData.py.. I would prefer to keep the changes to a minimum and revert this. The rule (enforced on a best effort basis) is to only run `black` on newly added modules (e.g. colorLib.builder), but keep whatever is the current style for older existing modules.

Reply:
Oops, will undo